tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -65,6 +68,13 @@ def test_get_dag_runs_success(self):
         self.assertEqual(data[0]['dag_id'], dag_id)
         self.assertEqual(data[0]['id'], dag_run.id)
 
+    @conf_vars({("core", "store_serialized_dags"): "True"})
+    def test_get_dag_runs_success_serialized_dags(self):
+        dagbag = DagBag(include_examples=True)
+        dag = dagbag.get_dag('example_bash_operator')
+        SerializedDagModel.write_dag(dag)
+        self.test_get_dag_runs_success()
```

Comment:
Can you extract the common part to the new method and call from two test methods?  This will make it easier to understand because tests should not depend on other tests.

Reply:
Done